.vscode/launch.json

```diff
@@ -2,12 +2,16 @@
     "version": "0.1.0",
```

Comment:
Don't think we need to track this config in the repo.

Reply:
Added to .gitignore.